src/main/scala/sample/exercises3/Exercises.scala

```diff
@@ -0,0 +1,49 @@
+package sample.exercises3
+
+import java.util.{Calendar, TimeZone}
+
+import scala.collection.mutable.{ArrayBuffer, ListBuffer}
+import util.control.Breaks._
+
+object Exercises {
+  def functional1(): Unit = {
+    println(TimeZone.getAvailableIDs.filter(_.contains("/")).map(_.split("/").tail).map(_ (0)).toList)
+  }
+//this is kinda slow
+  def primes(bound: Long): Unit = {
+    var count: Integer = 10
+    var num: Long = 1L
+    val t1 = System.nanoTime
+    for (i <- 2L to bound) {
+      breakable {
+        if (i % 2 == 0 || i % 3 == 0 || i % 5 == 0 || i % 7 == 0 || i % 11 == 0 || i % 13 == 0 || i % 17 == 0 || i % 19 == 0 || i % 23 == 0 || i % 29 == 0) break
```

Comment:
surely you could just create some function that would take in a collection with those elements: 2,3,5,7... and i and then give you a boolean back?